src/components/common/helpers/Helpers.tsx

```diff
@@ -1092,14 +1092,16 @@ export const clusterImageDescription = (nodeImageList: ImageList[], selectedImag
 export const processK8SObjects = (
     k8sObjects: ApiResourceType[],
     selectedResourceKind?: string,
+    nofilter?: boolean
```

Comment:
Please camel case this or change the name as discussed.

Reply:
done